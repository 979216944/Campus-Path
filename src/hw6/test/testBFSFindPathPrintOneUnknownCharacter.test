# Test BFS finding a path,
# it prints one "unknown character notFlower" line, if notFlower is not in graph.

# load graph
LoadGraph g flowers.tsv

# find a path with dest node no in the graph
FindPath g notFlower Aconitum