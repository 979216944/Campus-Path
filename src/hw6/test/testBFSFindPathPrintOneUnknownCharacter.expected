# Test BFS finding a path,
# it prints one "unknown character notFlower" line, if notFlower is not in graph.

# load graph
loaded graph g

# find a path with dest node no in the graph
unknown character notFlower